returnn/dataset.py

```diff
@@ -8,7 +8,7 @@
 
 import numpy
 
-from i6_core.returnn.config import ReturnnConfig
```

Comment:
Why this change?

Reply:
It got in as a change since we added the helper here, but when reverting the changes I found this to be better since we have the init file, but I can change it back if you want.